.github/workflows/ci.yml

```diff
@@ -83,7 +83,7 @@ jobs:
           - "7.4"
 
         php-ini-values:
-          - assert.exception=1, zend.assertions=1
+          - assert.exception=1, zend.assertions=1, phar.readonly=false
```

Comment:
Ah, right, GitHub actions makes this really easy. Awesome.

Reply:
Not so easy as I expected. Have you got any idea why it had no effect on PHP 7.1 and 7.4?